### PR TITLE
Mention VSCode's Command Palette in "Troubleshooting"

### PR DESCRIPTION
**Pull request by vi**
Resolves https://github.com/rust-analyzer/rust-analyzer/issues/11507

### CODE REVIEW

File: manual.adoc [hunk 1]
```diff
@@ -430,7 +430,7 @@ https://gitlab.com/cppit/jucipp[juCi++] has built-in support for the language se
 == Troubleshooting
 
 Start with looking at the rust-analyzer version.
-Try **Rust Analyzer: Show RA Version** in VS Code and `rust-analyzer --version` in the command line.
+Try **Rust Analyzer: Show RA Version** in VS Code (using **Command Palette** feature typically acticated by Ctrl+Shift+P) or `rust-analyzer --version` in the command line.
```
Comment (edwin0cheng) [inline manual.adoc:433]: acticated ?

Comment (vi) [reply manual.adoc:433]: "activated".

**Comment by lnicola** [comment]
Sorry, can you file this against the main RA repo?

**Comment by vi** [comment]
@lnicola , [Done](https://github.com/rust-analyzer/rust-analyzer/pull/11645).

**Pull request closed without merging**